packages/docs/community/release-process/smart-contracts.md

```diff
@@ -101,7 +101,8 @@ Deploying a new contract release should occur with the following process:
         <li>Receive report from auditors.</li>
         <li>If all issues in the audit report have straightforward fixes, announce a forthcoming smart contract release. </li>
         <li>Commit audit fixes to <code>master</code> and cherry-pick to the release branch.</li>
-        <li>Submit audit fixes to auditors for review. </li>
+        <li>Submit audit fixes to auditors for review.</li>
+        <li>Let the community know about the upcoming release proposal by posting details to the Governance category on https://forum.celo.org and cross post in the Discord <code>#governance</code> channel. See the 'Communication guidelines' section below for information on what your post should contain.</li>
```

Comment:
maybe add link to Discord channel here.

Reply:
I tried but can't add links in an md table